ArduCopter/avoidance_adsb.cpp

```diff
@@ -144,6 +144,16 @@ void AP_Avoidance_Copter::set_mode_else_try_RTL_else_LAND(Mode::Number mode)
     }
 }
 
+int16_t AP_Avoidance_Copter::get_altitude_minimum() const
+{
+    int16_t altitude = 0;
```

Comment:
maybe
```suggestion
#if MODE_RTL_ENABLED == ENABLED
    // do not descend if below RTL alt
    return copter.g.rtl_altitude;
else
    return 0;
#endif
```

as we don't really need the variable and we should let the complier optimise this

Reply:
Thanks! I fixed it.